client/blocks/image-editor/image-editor-toolbar.jsx

```diff
@@ -66,6 +75,24 @@ class ImageEditorToolbar extends Component {
 	onAspectOpen( event ) {
 		event.preventDefault();
 
+		const { imageMeetsMinimumDimensions, translate } = this.props;
+
+		if ( imageMeetsMinimumDimensions === false ) {
+			const noticeText = translate(
+				'To change aspect ratio, your image dimensions should be greater than %(width)px wide ' +
```

Comment:
Your earlier component example looks like it should work, but you'll likely need to update this first.